fluid/language_model/train.py

```diff
@@ -138,11 +164,19 @@ def train(train_reader,
     print("finish training")
 
 
+def get_cards():
+    cards = os.environ.get('CUDA_VISIBLE_DEVICES')
+    num = len(cards.split(","))
+    return num
+
+
 def train_net():
     """ do training """
     batch_size = 20
+    args = parse_args()
     vocab, train_reader, test_reader = utils.prepare_data(
-        batch_size=batch_size, buffer_size=1000, word_freq_threshold=0)
+        batch_size=batch_size * get_cards(), buffer_size=1000, \
```

Comment:
It seems that `get_cards()` can only work under CE(with `CUDA_VISIBLE_DEVICES` setting)，please check the usage in situation other than CE.